bitnami/postgresql/templates/statefulset-slaves.yaml

```diff
@@ -95,6 +95,15 @@ spec:
               {{- if and .Values.shmVolume.enabled .Values.shmVolume.chmod.enabled }}
               chmod -R 777 /dev/shm
               {{- end }}
+              {{- if .Values.tls.enabled }}
+              cp /tmp/certs/* /opt/bitnami/postgresql/certs/
```

Comment:
Is it guaranteed that there will be a file inside that folder? Note that dotfiles (`.file`) and files inside a subdirectory will not be copied with this `cp` command.

Reply:
Yes, when `.Values.tls.enabled=true` we have some templates in `_helpers.tpl` than enforce the existence of cert files. 

E.g.:

```yaml
{{- define "postgresql.tlsCert" -}}
{{- required "Certificate filename is required when TLS in enabled" .Values.tls.certFilename | printf "/opt/bitnami/postgresql/certs/%s" -}}
{{- end -}}
```

Those cert files are mounted in `/tmp/certs/` in the init-container. Regarding dotfiles, the main goal of this init-container workaround is to set the right permissions to `.crt`, `.key` and `.crl` files, so no issues should be expected with respect to that.